services/director/src/simcore_service_director/cache_request_decorator.py

```diff
@@ -6,25 +6,25 @@
 
 from simcore_service_director import config
 
-def cache_requests(http_request):
+
+def cache_requests(http_request: callable):
```

Comment:
``typing.Callable`` instead of callable, oder? See https://docs.python.org/3/library/typing.html#typing.Callable

Reply:
right.